msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/GetAccessTokenError.kt

```diff
@@ -27,6 +27,10 @@ import com.microsoft.identity.nativeauth.statemachine.results.GetAccessTokenResu
 
 internal class GetAccessTokenErrorTypes {
     companion object {
+        /*
+         * The NO_ACCOUNT_FOUND value indicates the user is not signed in.
+         * If this occurs, the API should be called after successful sign in
+         */
```

Comment:
`NO_ACCOUNT_RECORD` should have been in the Error.kt file but cannot be removed now without breaking backward compatibility.

Reply:
Why should it have been in Error.kt? AFAIK that class is only for common/shared errors, which this is not right? E.g. we've also got `SignUpErrorTypes`. 
Moreover, all error type classes such as this one should be `internal`, so developers can't access them. They should be using the utility methods, or do raw String inspection if they want to. So AFAIK moving this wouldn't be a breaking change.

Reply:
I have moved the variable back to `GetAccessTokenErrorTypes` class